File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLRadarGraphProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLRadarGraphProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String MaxValue;
        public String ColorStr;
        public String NumMarks;
        public String Title;
        public String TitleTextColor;
        public String TitleTextHeight;
        public String TitleTextFontString;
        public String H;
        public String MarkLabelTextColor;
        public String MarkLabelTextHeight;
        public String MarkLabelTextFontString;
        public String AlreadyUnregisteredAnimation;
        public Object Tag;
        public String TabStopIndex;

        CCLRadarGraphProps()
        {
            Data = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLRadarGraphProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLRadarGraphProps
    {
        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<float> Data { get; set; }
        public int MaxValue { get; set; }
        public string ColorStr { get; set; }
        public int NumMarks { get; set; }
        public string Title { get; set; }
        public string TitleTextColor { get; set; }
        public int TitleTextHeight { get; set; }
        public string TitleTextFontString { get; set; }
        public int H { get; set; }
        public string MarkLabelTextColor { get; set; }
        public int MarkLabelTextHeight { get; set; }
        public string MarkLabelTextFontString { get; set; }
        public int AlreadyUnregisteredAnimation { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLRadarGraphProps()
        {
            Data = new List<float>();
        }
    }
}